src/pages/guides/journaling-intro.md

```diff
@@ -4,21 +4,22 @@ title: Journaling Introduction
 
 # Journaling
 
-For enterprise developers, Adobe offers another way to consume events besides webhooks: journaling. 
-The Adobe I/O Events Journaling API enables enterprise integrations to consume events according 
-to their own cadence and process them in bulk. 
-Unlike webhooks, no additional registration or other configuration is required; 
-every enterprise integration that is registered for events is automatically enabled for journaling. 
-Journaling data is retained for 7 days.
+The Adobe I/O Events [Journaling API](./api/journaling-api.md) allows enterprise integrations to retrieve events at their own pace and process them in bulk.
+For enterprise developers, Adobe provides an alternative method for consuming events beyond all push-based deliveries ([Webhooks](./index.md), [Amazon EventBridge](./amazon-eventbridge/index.md), and [Runtime](./runtime-webhooks/index.md)): **Journaling**.
```

Comment:
this is unrelated, right ? 
why mentioning webhook and eventbridge here ?

Reply:
Yes it is unrelated. I just wanted to stress on the fact that all of these push deliveries are supported today